fw/http_frame.c

```diff
@@ -1894,21 +1894,25 @@ tfw_h2_calc_frame_length(TfwH2Ctx *ctx, TfwStream *stream, TfwFrameType type,
 static inline char
 tf2_h2_calc_frame_flags(TfwStream *stream, TfwFrameType type)
```

Comment:
please rename this functiin to `tf2_h2_calc_frame_flags` it was my mistake, I miss it in my PR

Reply:
what do you mean? It's already named as `tf2_h2_calc_frame_flags`.

Reply:
tfw_h2_calc_frame_flags sorry again my mistake)